.regal/config.yaml

```diff
@@ -7,7 +7,9 @@ rules:
     file-length:
       level: warning
     line-length:
-      max-line-length: 130
+      level: warning
+    rule-length:
+      max-rule-length: 50
```

Comment:
what?! :sweat_smile:

Reply:
oh this is rule length, not line length.

Reply:
lol. The default is 30. Seemed just as arbitrary as 50.